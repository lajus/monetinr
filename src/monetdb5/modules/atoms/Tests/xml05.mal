b:= bat.new(:oid,:str);
bat.insert(b,0@0,"Mary Ann Walker");
bat.insert(b,1@0,"Annie Chapman");
h:= bat.new(:oid,:str);
bat.insert(h,0@0,"brown");
bat.insert(h,1@0,"curly dark brown");

io.print(b);

bs:= xml.xml(b);
bc:= xml.element("name",bs);

aa:= xml.root(bc,"1.0","no");
as:= xml.str(aa);
io.print(as);
