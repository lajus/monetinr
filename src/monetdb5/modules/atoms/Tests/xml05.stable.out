stdout of test 'xml05` in directory 'monetdb5/modules/atoms` itself:


# 07:41:18 >  
# 07:41:18 >  Mtimeout -timeout 60  mserver5 "--config=/ufs/mk/monet5/Linux/etc/monetdb5.conf" --debug=10 --set "monet_mod_path=/ufs/mk/monet5//Linux/lib/MonetDB5:/ufs/mk/monet5//Linux/lib/MonetDB5/lib:/ufs/mk/monet5//Linux/lib/MonetDB5/bin" --set "gdk_dbfarm=/ufs/mk/monet5//Linux/var/MonetDB5/dbfarm" --set "sql_logdir=/ufs/mk/monet5//Linux/var/MonetDB5/sql_logs" --set "xquery_logdir=/ufs/mk/monet5//Linux/var/MonetDB5/xquery_logs" --set mapi_open=true --set xrpc_open=true --set mapi_port=32400 --set xrpc_port=45758 --set monet_prompt= --trace --dbname=mTests_src_modules_mal  xml05.mal
# 07:41:18 >  

# MonetDB server v5.1.0, based on kernel v1.19.0
# Serving database 'mTests_src_modules_mal'
# Compiled for x86_64-redhat-linux-gnu/64bit with 64bit OIDs dynamically linked
# Copyright (c) 1993-2007 CWI, all rights reserved
# Visit http://monetdb.cwi.nl/ for further information
# Listening for connection requests on mapi:monetdb://eir.ins.cwi.nl:34949/
# MonetDB/SQL module v2.39.0 loaded
# MonetDB/GIS module v0.19.0 loaded

function user.main():void;
    b := bat.new(:oid,:str);
    bat.insert(b,0@0,"Mary Ann Walker");
    bat.insert(b,1@0,"Annie Chapman");
    h := bat.new(:oid,:str);
    bat.insert(h,0@0,"brown");
    bat.insert(h,1@0,"curly dark brown");
    io.print(b);
    bs := xml.xml(b);
    bc := xml.element("name",bs);
    aa := xml.root(bc,"1.0","no");
    as := xml.str(aa);
    io.print(as);
end main;
#---------------------------------#
# h	t			  # name
# void	str			  # type
#---------------------------------#
[ 0@0,	  "Mary Ann Walker"	  ]
[ 1@0,	  "Annie Chapman"	  ]
#-----------------------------------------------------------------------------------------#
# h	t										  # name
# void	str										  # type
#-----------------------------------------------------------------------------------------#
[ 0@0,	  "<?xml version=\"1.0\" standalone=\"no\"?><name>Mary Ann Walker</name>"	  ]
[ 1@0,	  "<?xml version=\"1.0\" standalone=\"no\"?><name>Annie Chapman</name>"		  ]

# 07:41:18 >  
# 07:41:18 >  Done.
# 07:41:18 >  

